docs/developers/celestia-node.md

```diff
@@ -13,16 +13,20 @@ cd $HOME
 rm -rf celestia-node
 git clone https://github.com/celestiaorg/celestia-node.git
 cd celestia-node/
+NODE_VERSION=$(curl -s \ 
+   https://api.github.com/repos/celestiaorg/celestia-node/releases/latest \ 
+   | jq -r ".tag_name") 
+git checkout tags/$NODE_VERSION -b $NODE_VERSION
 make install
 ```
 
 Verify that the binary is working and check the version with `celestia version` command:
 
 ```console
 $ celestia version
-Semantic version: v0.2.0
-Commit: 1fcf0c0bb5d5a4e18b51cf12440ce86a84cf7a72
-Build Date: Fri 04 Mar 2022 01:15:07 AM CET
+Semantic version: v0.3.0-rc1
+Commit: 5843c5e102651206fe11c685c163f267b141b103
+Build Date: Thu 26 May 2022 02:32:58 PM CEST
```

Comment:
optional not: It might not be worth mentioning Build Date as it is unique to each build

Reply:
removed the bottom 3 lines as theyre not really important